arch/arm/Kconfig

```diff
@@ -19,4 +19,11 @@ config ARCH_DEFCONFIG
 	string
 	default "arch/arm/defconfig"
 
+config CPU_HAS_FPU_DP
```

Comment:
This option does not have a nomenclature that is specific to ARM. So, it should better be in the parent Kconfig file.